### ISSUE TITLE
The contacts wouldn't load in the web interface after an upgrade

### ISSUE THREAD

**Issue by dniku** [owncloud/contacts#359]
### Steps to reproduce
1. On Debian Jessie, perform the following upgrade (copied from `/var/log/apt/history.log`): `Upgrade: tzdata:armhf (2015g-1, 2016c-0+deb8u1), owncloud-deps-php5:armhf (9.0.0-1.1, 9.0.0-2.1), owncloud:armhf (9.0.0-1.1, 9.0.0-2.1), owncloud-files:armhf (9.0.0-1.1, 9.0.0-3.1)`
2. Perform `occ upgrade` and disable maintenance mode by editing `config/config.php` manually.
3. Go to the contacts app.
4. The contacts never load:

![image](https://cloud.githubusercontent.com/assets/1315874/14532776/efcd42f6-026a-11e6-85ad-9dc96577b4e2.png)
### Server configuration

**Operating system**: fully upgraded Debian Jessie ARM:

```
$ uname -srv
Linux 3.10.80 #1 SMP PREEMPT Sat Jun 20 10:47:49 GMT 2015
```

**Web server:** `Server version: Apache/2.4.17 (Debian)`

**Database:** `mysql  Ver 15.1 Distrib 10.0.23-MariaDB, for debian-linux-gnueabihf (armv7l) using readline 5.2`

**PHP version:**

```
$ php --version
PHP 5.6.19-0+deb8u1 (cli) (built: Mar 25 2016 05:58:21) 
Copyright (c) 1997-2016 The PHP Group
Zend Engine v2.6.0, Copyright (c) 1998-2016 Zend Technologies
    with Zend OPcache v7.0.6-dev, Copyright (c) 1999-2016, by Zend Technologies
```

**ownCloud version:** ownCloud 9.0.1 (stable)

**Contacts version:** 1.1.0.0

**Updated from an older ownCloud or fresh install:** an old install, was v7 initially, then v8, now v9

```
Login as admin user into your ownCloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.

No errors have been found.
```

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your ownCloud installation folder

$ sudo -u www-data php occ app:list
Enabled:
  - activity: 2.2.1
  - calendar: 1.1
  - comments: 0.2
  - contacts: 1.1.0.0
  - dav: 0.1.6
  - federatedfilesharing: 0.1.0
  - federation: 0.0.4
  - files: 1.4.4
  - files_pdfviewer: 0.8
  - files_sharing: 0.9.1
  - files_texteditor: 2.1
  - files_trashbin: 0.8.0
  - files_versions: 1.2.0
  - files_videoplayer: 0.9.8
  - firstrunwizard: 1.1
  - gallery: 14.5.0
  - notifications: 0.2.3
  - provisioning_api: 0.4.1
  - systemtags: 0.2
  - tasks: 0.9.0
  - templateeditor: 0.1
  - updatenotification: 0.1.0
Disabled:
  - encryption
  - external
  - files_external
  - user_external
  - user_ldap
```

**The content of config/config.php:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your ownCloud installation folder

{
    "system": {
        "instanceid": "** EDITED **",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            ** EDITED **
        ],
        "datadirectory": "** EDITED **",
        "overwrite.cli.url": "https:\/\/localhost",
        "dbtype": "mysql",
        "version": "9.0.1.3",
        "installed": true,
        "forcessl": true,
        "mail_smtpmode": "php",
        "loglevel": 0,
        "preview_libreoffice_path": "\/usr\/bin\/libreoffice",
        "theme": "",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "mail_from_address": "** EDITED **",
        "mail_domain": "** EDITED **",
        "updatechecker": false,
        "dbname": "owncloud",
        "dbhost": "localhost",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "trashbin_retention_obligation": "auto",
        "maintenance": false
    }
}
```

**Are you using external storage, if yes which one:** only `/data` on an external HDD

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** no
### Client configuration

**Browser:** Versión 49.0.2623.110 (64-bit)

**Operating system:** Gentoo Linux

```
$ uname -srv
Linux 4.1.15-gentoo-r1 #6 SMP Tue Mar 1 15:44:57 MSK 2016
```
#### Web server error log

There is nothing in `error.log`.

`access.log`:

```
192.168.1.1 - - [14/Apr/2016:18:12:50 +0300] "GET /owncloud/index.php/apps/contacts HTTP/1.1" 200 6865 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:50 +0300] "GET /owncloud/index.php/apps/contacts HTTP/1.1" 200 6865 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/css/public/style.css?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 2933 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/css/public/style.css?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 2933 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/js/vendor/angular-route/angular-route.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 10657 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/js/vendor/angular-route/angular-route.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 10657 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/js/vendor/angular-uuid4/angular-uuid4.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 1717 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/js/vendor/angular-uuid4/angular-uuid4.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 1717 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/js/vendor/angular-cache/dist/angular-cache.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 8423 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/js/vendor/angular-cache/dist/angular-cache.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 8423 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/js/vendor/angular-sanitize/angular-sanitize.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 8743 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/js/vendor/angular-sanitize/angular-sanitize.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 8743 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/js/vendor/vcard/src/vcard.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 2780 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/js/vendor/vcard/src/vcard.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 2780 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/js/vendor/ui-select/dist/select.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 20973 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/js/vendor/ui-select/dist/select.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 20973 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/js/vendor/angular-bootstrap/ui-bootstrap.min.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 26647 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/js/vendor/angular-bootstrap/ui-bootstrap.min.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 26647 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/js/vendor/angular-bootstrap/ui-bootstrap-tpls.min.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 30450 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/js/vendor/angular-bootstrap/ui-bootstrap-tpls.min.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 30450 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/js/public/script.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 36462 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/js/public/script.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 36462 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/js/vendor/jquery-timepicker/jquery.ui.timepicker.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 15011 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/js/vendor/jquery-timepicker/jquery.ui.timepicker.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 15011 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/js/dav/dav.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 51287 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/js/dav/dav.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 51287 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/index.php/core/js/oc.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 3712 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/index.php/core/js/oc.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 3712 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/js/vendor/angular/angular.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 280456 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/apps/contacts/js/vendor/angular/angular.js?v=fa366d9190a2621c3420acd14245744b HTTP/1.1" 200 280456 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/index.php/avatar/**USERNAME**/32 HTTP/1.1" 200 1817 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:51 +0300] "GET /owncloud/index.php/avatar/**USERNAME**/32 HTTP/1.1" 200 1817 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
192.168.1.1 - - [14/Apr/2016:18:12:52 +0300] "GET /owncloud/apps/calendar/img/calendar.svg HTTP/1.1" 304 195 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:52 +0300] "GET /owncloud/apps/calendar/img/calendar.svg HTTP/1.1" 304 195 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
192.168.1.1 - - [14/Apr/2016:18:12:52 +0300] "GET /owncloud/ocs/v2.php/apps/notifications/api/v1/notifications?format=json HTTP/1.1" 200 933 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:52 +0300] "GET /owncloud/ocs/v2.php/apps/notifications/api/v1/notifications?format=json HTTP/1.1" 200 933 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
192.168.1.1 - - [14/Apr/2016:18:12:52 +0300] "GET /owncloud/remote.php/dav/addressbooks HTTP/1.1" 200 999 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:52 +0300] "GET /owncloud/remote.php/dav/addressbooks HTTP/1.1" 200 999 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
192.168.1.1 - - [14/Apr/2016:18:12:53 +0300] "PROPFIND /owncloud/remote.php/dav/addressbooks HTTP/1.1" 207 1642 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:53 +0300] "PROPFIND /owncloud/remote.php/dav/addressbooks HTTP/1.1" 207 1642 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
192.168.1.1 - - [14/Apr/2016:18:12:53 +0300] "PROPFIND /owncloud/remote.php/dav/principals/users/**USERNAME**/ HTTP/1.1" 207 1665 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:53 +0300] "PROPFIND /owncloud/remote.php/dav/principals/users/**USERNAME**/ HTTP/1.1" 207 1665 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
192.168.1.1 - - [14/Apr/2016:18:12:53 +0300] "PROPFIND /owncloud/remote.php/dav/addressbooks/users/**USERNAME**/ HTTP/1.1" 207 2439 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:53 +0300] "PROPFIND /owncloud/remote.php/dav/addressbooks/users/**USERNAME**/ HTTP/1.1" 207 2439 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
192.168.1.1 - - [14/Apr/2016:18:12:53 +0300] "PROPFIND /owncloud/remote.php/dav/addressbooks/users/**USERNAME**/Contacts/ HTTP/1.1" 207 216212 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:53 +0300] "PROPFIND /owncloud/remote.php/dav/addressbooks/users/**USERNAME**/Contacts/ HTTP/1.1" 207 216212 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
192.168.1.1 - - [14/Apr/2016:18:12:54 +0300] "PROPFIND /owncloud/remote.php/dav/addressbooks/users/**USERNAME**/ HTTP/1.1" 207 1545 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:54 +0300] "PROPFIND /owncloud/remote.php/dav/addressbooks/users/**USERNAME**/ HTTP/1.1" 207 1545 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
192.168.1.1 - - [14/Apr/2016:18:12:54 +0300] "REPORT /owncloud/remote.php/dav/addressbooks/users/**USERNAME**/Contacts/ HTTP/1.1" 207 177754 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
**DOMAIN**:443 192.168.1.1 - - [14/Apr/2016:18:12:54 +0300] "REPORT /owncloud/remote.php/dav/addressbooks/users/**USERNAME**/Contacts/ HTTP/1.1" 207 177754 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/49.0.2623.110 Safari/537.36"
```
#### ownCloud log (data/owncloud.log)

There is nothing relevant in `owncloud.log`.
#### Browser log

Chrome console log:

```
Uncaught (in promise) TypeError: Cannot read property 'value' of undefined(…)    contact_model.js:18
Uncaught (in promise) TypeError: Cannot read property 'value' of undefined(…)    contact_model.js:18
```

The full network log from Chrome is very large and contains a lot of sensitive data. I will script something up to clean it up if you need it, but there do not appear to be any errors.

**Comment by DeepDiver1975** [comment]
Most probably there is one vcard holding a character which is causing issues on the transport layer of the app

**Comment by dniku** [comment]
I'm not sure if this is relevant, but Thunderbird with SoGo is unable to sync the contacts with a proper encoding. [The ArtLebedev decoder](http://www.artlebedev.ru/tools/decoder/) thinks that whatever Thunderbird displays can be decoded as `7-bit ASCII → ISO-8859-5`. This produces a result that is readable but is not correct (the lowercase letters become uppercase and uppercase letters become garbage).

**Comment by dethos** [comment]
I'm not sure if related, but after an upgrade my contacts do not load as described in this issue (I get the same blank page), however the i get the following in the browser console log:

> 15:47:42.403 SyntaxError: missing ; before statement - angular.js:309:9
> 15:47:42.413 TypeError: angular is undefined - angular-route.js:1:54
> 15:47:42.418 ReferenceError: angular is not defined - angular-uuid4.js:12:1
> 15:47:42.426 TypeError: angular is undefined - angular-cache.js:8:136
> 15:47:42.432 TypeError: angular is undefined - angular-sanitize.js:1:54
> 15:47:42.444 ReferenceError: angular is not defined - select.js:2:279

In my case the desktop clients seem to work as they would normally.

**Comment by pbek** [comment]
I also upgraded (from ownCloud 8.2.x) to ownCloud 9.0.1 with Contacts 1.1.0.0 and get the same "empty" page.
When I load the contact page I only get some errors like this in the browser console:

```
contact_model.js:18 Uncaught (in promise) TypeError: Cannot read property 'value' of undefined(…)Contact.angular.extend.uid @ contact_model.js:18(anonymous function) @ contact_service.js:36
contact_model.js:18 Uncaught (in promise) TypeError: Cannot read property 'value' of undefined
    at Contact.angular.extend.uid 
```

...no errors in the `owncloud.log`.
@DeepDiver1975 is there a way to find out if there are broken calendar objects that might cause problems? I have about 2700 calendar objects.

**Comment by DeepDiver1975** [comment]
> @DeepDiver1975 is there a way to find out if there are broken calendar objects that might cause problems? I have about 2700 calendar objects.

I assume you mean contacts/vcard - right? :wink:

**Comment by pbek** [comment]
> I assume you mean contacts/vcard - right?

Sorry, you are right, we are talking about vcards. Although I also have troubles with the Tasks app. ;) Calendars seem to work.
Do you have any insights about the calendar problem or a way to produce some meaningful log messages by the dav backend?

**Comment by DeepDiver1975** [comment]
> some meaningful log messages by the dav backend?

The dav backend is most probably not causing any issues - I know of issues in the client side lib used in the contacts app where the full vcard is not received by the application code because of one char causing issues .... we need to analyse this in depth.

Maybe we add a temporary work around to ignore that empty card and continue processing ....

**Comment by pbek** [comment]
That would be awesome! :tada:

**Comment by DeepDiver1975** [comment]
address with #374

**Comment by dniku** [comment]
This is still broken on Debian with the version `owncloud:armhf (9.0.1-3.1)`. Did this fix get shipped in that version?

**Comment by DeepDiver1975** [comment]
> This is still broken on Debian with the version owncloud:armhf (9.0.1-3.1). Did this fix get shipped in that version?

the fix is in the conatcts app 1.2 - did you update the app?

**Comment by dniku** [comment]
Now I have and this bug is gone. Thanks.
